lib/KToolbar.vue

```diff
@@ -1,328 +0,0 @@
-          <UiIconButton
-            size="large"
-            type="secondary"
-            :color="textColor"
-            :icon="navIcon"
-            @click="navIconClick"
-          />
-        </slot>
-      </div>
-
-      <div
-        v-if="brand || $slots.brand"
-        class="k-toolbar-brand"
-      >
-        <!-- @slot Customize the brand area. Shows brand text by default. -->
-        <slot name="brand">
-          <div class="k-toolbar-brand-text">
-            {{ brand }}
-          </div>
-        </slot>
-      </div>
-
-      <!-- @slot Main content area. Shows the title by default. -->
-      <slot>
-        <div
-          v-if="title"
-          class="k-toolbar-title"
-        >
-          {{ title }}
-        </div>
-      </slot>
-
-      <div
-        v-if="$slots.navigation"
-        class="k-toolbar-nav"
-      >
-        <!-- @slot Navigation links or menu items area. -->
-        <slot name="navigation"></slot>
-      </div>
-    </div>
-
-    <div class="k-toolbar-right">
-      <!-- @slot Action buttons or secondary controls area. -->
-      <slot name="actions"></slot>
-    </div>
-
-    <UiProgressLinear
-      v-show="loading"
-      class="k-toolbar-progress"
-      :color="progressColor"
-    />
-  </div>
-
-</template>
-
-
-<script>
-
-  import UiIconButton from './keen/UiIconButton.vue';
-  import UiProgressLinear from './keen/UiProgressLinear.vue';
-
-  export default {
-    name: 'KToolbar',
-    components: {
-      UiIconButton,
-      UiProgressLinear,
-    },
-
-    props: {
-      /**
-       * Controls the toolbar appearance and styling. Options are:
-       * - 'default': Standard white background with shadow
-       * - 'colored': Uses theme primary color as background
-       * - 'clear': Transparent background with no shadow or border
-       */
-      type: {
-        type: String,
-        default: 'default',
-        validator(value) {
-          return ['default', 'colored', 'clear'].includes(value);
-        },
-      },
-      /**
-       * Controls the text color throughout the toolbar. Use 'white' for dark backgrounds
-       * and 'black' for light backgrounds to ensure proper contrast.
-       */
-      textColor: {
-        type: String,
-        default: 'black',
-        validator(value) {
-          return ['black', 'white'].includes(value);
-        },
-      },
-      /**
-       * The main title text displayed in the toolbar
-       */
-      title: {
-        type: String,
-        default: '',
-      },
-      /**
-       * Brand text displayed alongside the title in the left area
-       */
-      brand: {
-        type: String,
-        default: '',
-      },
-      /**
-       * Icon name displayed in the navigation button when using the default icon slot
-       */
-      navIcon: {
-        type: String,
-        default: 'menu',
-      },
-      /**
-       * Whether to hide the navigation icon button completely
-       */
-      removeNavIcon: {
```

Comment:
Removed.

**Audit**
- Used in Kolibri from [AppBar](https://github.com/learningequality/kolibri/blob/ad8ebbdc43144f36d3d57d66d0e88f8c094d6853/packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue#L15) but doesn't take effect (whether the navigation icon is displayed or not is configured in [the #icon slot condition](https://github.com/learningequality/kolibri/blob/ad8ebbdc43144f36d3d57d66d0e88f8c094d6853/packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue#L31))